apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -91,6 +91,13 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
         const properties = Object.keys(document?.propertiesSchema?.properties ?? {}).reduce(
           (acc, curr) => {
             if (!formData?.[curr]) return acc;
+
+            if (document?.propertiesSchema?.properties?.[curr]?.format === 'date-time') {
+              if (formData?.[curr]?.length === 16) {
+                formData[curr] = `${formData[curr]}:00`;
+              }
+            }
+
```

Comment:
```suggestion
            if (document?.propertiesSchema?.properties?.[curr]?.format === 'date-time' && formData?.[curr]?.length === 16) {
                formData[curr] = `${formData[curr]}:00`;
            }
```

Reply:
`formData` here is React state. Mutating it is a bad practice and may behave in unexpected ways.